receiver/k8sclusterreceiver/factory.go

```diff
@@ -56,7 +64,20 @@ func createMetricsReceiver(
 	if err != nil {
 		return nil, err
 	}
-	return newReceiver(params.Logger, rCfg, consumer, k8sClient)
+
+	var osQuotaClient quotaclientset.Interface
+	switch rCfg.Distribution {
+	case distributionOpenShift:
+		osQuotaClient, err = rCfg.getOpenShiftQuotaClient()
+		if err != nil {
+			return nil, err
+		}
+	case distributionKubernetes:
```

Comment:
I think a comment like `// Nothing to initialize` could help to make clear the empty block isn't a typo